docs/_docs/02_features/auto-init.md

```diff
@@ -27,6 +27,6 @@ As [mentioned]({{site.baseurl}}/docs/features/keep-your-cli-flags-dry/#extra_arg
 
 Note that there might be cases where terragrunt does not properly detect that `terraform init` needs be called. In this case, terraform would fail. Running `terragrunt init` again corrects this situation.
 
-For some use cases, it might be desirable to disable Auto-Init. For example, if each user wants to specify a different `-plugin-dir` option to `terraform init` (and therefore it cannot be put in `extra_arguments`). To disable Auto-Init, use the `--terragrunt-no-auto-init` command line option or set the `TERRAGRUNT_AUTO_INIT` environment variable to `false`.
+For some use cases, it might be desirable to disable Auto-Init. For example, if each user wants to specify a different `-plugin-dir` option to `terraform init` (and therefore it cannot be put in `extra_arguments`). To disable Auto-Init, use the `--terragrunt-no-auto-init` command line option or set the `TERRAGRUNT_NO_AUTO_INIT` environment variable to `true`.
```

Comment:
Did we remove support for the prev env var? So this is a breaking change?

Reply:
No. I've added backwards compatibility for these env vars, and explained why I did it [here](https://github.com/gruntwork-io/terragrunt/pull/2608#issuecomment-1631348902)

https://github.com/gruntwork-io/terragrunt/blob/7f242e9ffbfe8b8f773fa8fe59d7303e8c9f8793/cli/app.go#L115-L118